### PR TITLE
w2_d5 quiz

### PR DESCRIPTION
**Pull request by maggiemccain**

### CODE REVIEW

File: Maggie_McCain/quiz/quiz.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+//(1.) Write the correct javascript to select the second h1 element:
+var secondTitle = document.getElementsByTagName('h1')[1];
```
Comment (jordanetzler) [inline Maggie_McCain/quiz/quiz.js:2]: WOOO!!!! IT'S RIGHT!!!!!!!

File: Maggie_McCain/quiz/quiz.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+//(2.) How would you change the contents of the element to be 'Friend'?
```
Comment (jordanetzler) [inline Maggie_McCain/quiz/quiz.js:4]: yeah that works

File: Maggie_McCain/quiz/quiz.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+var clickMeBtn = document.getElementsByTagName('button')[0];
+
+function divBackgroundCol() {
+  var div = document.getElementsByTagName('div')[0];
+  div.style.backgroundColor = 'red';
+}
+
+clickMeBtn.addEventListener('click', divBackgroundCol);
+
```
Comment (jordanetzler) [inline Maggie_McCain/quiz/quiz.js:39]: Good work young grasshopper!

File: Maggie_McCain/quiz/quiz.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+//
+//   </div>
+//   </body>
+//
+// </html>
+// ```
+var myArticlesDiv = document.getElementsByClassName('my-articles')[0];
+var paragraph = document.createElement('p');
+myArticlesDiv.appendChild(paragraph);
+
```
Comment (jordanetzler) [inline Maggie_McCain/quiz/quiz.js:69]: Damn! yeah right too

File: Maggie_McCain/quiz/quiz.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+//     <li></li>
+//     <li></li>
+//     <li></li>
+//     <li></li>
+//   </ul>
+//   </body>
+//
+// </html>
+var listItems = document.getElementsByTagName('li');
+listItems.length;
```
Comment (jordanetzler) [inline Maggie_McCain/quiz/quiz.js:90]: PERFECT! Your a star 5.98 out of 5. I guess you could round up :)

**Comment by Kanairy** [comment]
Maggie, good work on the quiz! Before I merge, can you please move the quiz folder into a folder named w2_d5 and make another commit and push?

**Pull request merged**